tests/tck/utils/assertions.py

```diff
@@ -189,13 +189,18 @@ def assert_statistics(resultset, stat, value):
         Env.RTestInstance.currEnv.assertEquals(resultset.nodes_created, value)
     elif stat == "+relationships":
         Env.RTestInstance.currEnv.assertEquals(resultset.relationships_created, value)
+    elif stat == "-relationships":
+        Env.RTestInstance.currEnv.assertEquals(resultset.relationships_deleted, value)
     elif stat == "+labels":
         Env.RTestInstance.currEnv.assertEquals(resultset.labels_added, value)
     elif stat == "+properties":
         Env.RTestInstance.currEnv.assertEquals(resultset.properties_set, value)
     elif stat == "-nodes":
         Env.RTestInstance.currEnv.assertEquals(resultset.nodes_deleted, value)
+    elif stat == "-labels" or stat == "-properties":
+        pass
```

Comment:
This has the potential of hiding incorrectness

Reply:
we don't report about this stats
in the Scala check this is not checked by stats returned from the query